acpt.php

```diff
@@ -7,15 +7,14 @@
 Author URI: http://iambrian.com/
 */
 
-if ( ! defined( 'ABSPATH' ) )
```

Comment:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style

Usually, you want to use a new line.  On short things like this, I'll usually drop it all in a single line though. Preference here for the most part.